static/app/utils/replays/replayReader.tsx

```diff
@@ -124,6 +186,67 @@ export default class ReplayReader {
     return this.replayRecord;
   };
 
+  getRRWebFrames = () => this._rrwebEvents;
+
+  getSortedRRWebFrames = memoize(() =>
+    this.getRRWebFrames().sort((a, b) => a.timestamp - b.timestamp)
+  );
+
+  getConsoleFrames = memoize(() =>
+    this._breadcrumbFrames.filter(frame => frame.category === 'console')
+  );
+
+  getNetworkFrames = memoize(() =>
+    this._spanFrames.filter(
+      frame => frame.op.startsWith('navigation.') || frame.op.startsWith('resource.')
+    )
+  );
+
+  getDOMFrames = memoize(() =>
+    this._breadcrumbFrames.filter(frame => 'nodeId' in (frame.data ?? {}))
+  );
+
+  getMemoryFrames = memoize(() =>
+    this._spanFrames.filter(frame => frame.op === 'memory')
+  );
+
+  _getChapters = () => [
+    ...this._breadcrumbFrames.filter(frame =>
+      ['replay.init', 'ui.click', 'replay.mutations', 'ui.slowClickDetected'].includes(
+        frame.category
+      )
+    ),
```

Comment:
This is going to filter out custom frames now, so like "Redux Action" for example won't have a home anymore.

We can put it back in here for sure, but maybe it's also a good time to ask where the best spots are for it to be.

Reply:
Also this hides the different dead/slow/rage click events! https://github.com/getsentry/sentry/issues/50590

Reply:
I think we should leave custom breadcrumbs in instead of taking it away. I know some users use the redux crumbs and/or custom crumbs and expect them to be there. I know we have one request for crumbs to be searchable and in the timeline even.

I do agree we should start to think about how to treat them.

#50590 says we should have the dead/rage clicks visible for us too

Reply:
ya. i sort of noticed that the custom ones were removed after the fact. fixing now!